.betterer.ts

```diff
@@ -75,17 +72,14 @@ function regexp(pattern: RegExp, issueMessage: string) {
 }
 
 function countEslintErrors() {
-  return new BettererFileTest(async (filePaths, fileTestResult, resolver) => {
+  return new BettererFileTest(async (filePaths, fileTestResult) => {
     // Just bail early if there's no files to test. Prevents trying to get the base config from failing
     if (filePaths.length === 0) {
       return;
     }
 
-    const { baseDirectory } = resolver;
-
     const runner = new ESLint({
-      overrideConfig: config,
```

Comment:
Overriding this way seems to mean that it doesn't extend the config, and instead uses the betterer config in isolation